packages/client/lib/rpc/modules/engine.ts

```diff
@@ -511,7 +511,7 @@ export class Engine {
             headBlock.hash()
           )}`
         )
-        this.service.beaconSync?.setHead(headBlock)
+        await this.service.beaconSync?.setHead(headBlock)
```

Comment:
Oh, linting already at our service (or is this no rule and you side-discovered?) 🙂

Reply:
But it wasn't added in this PR or was it (can't find)? (just out of interest, wonder how this was triggered?)

Reply:
Ah, then it's likely some bug in their parser on how to find this locations I guess.

Reply:
Thanks a lot for the upgrade here, just merged! 🙂